src/pluto/src/http.rs

```diff
@@ -388,4 +388,4 @@ impl HttpServe {
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
let's leave empty line at end of the file
```suggestion
}

```